src/platforms/hosted/cmsis_dap.c

```diff
@@ -413,7 +413,7 @@ ssize_t dbg_dap_cmd_bulk(const uint8_t *const request_data, const size_t request
 	const size_t response_length)
 {
 	int transferred = 0;
-	const int result = libusb_bulk_transfer(
+	int result = libusb_bulk_transfer(
```

Comment:
Perhaps, rather than removing const here, this becomes `request_result` and the one below `response_result`? It would make us feel more comfortable to not be loosing on const-correctness with this code.